### DISCUSSION TITLE
[Enhancement] [table] 为什么某些组件（如table）的默认字号不受common/vars中的变量控制，而是写死在table.scss中

### DISCUSSION THREAD

**Discussion by reallyreallycai** [element-plus/element-plus#12259] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

table.scss中写死了font-size为14px，通过forward重新设置$table-font-size只在为table设置size时生效，为什么表格默认字号不考虑用common/vars中的$font-size来设置呢？
table.scss:
@include b(table) {
  position: relative;
  overflow: hidden;
  box-sizing: border-box;
  height: fit-content;
  width: 100%;
  max-width: 100%;
  background-color: getCssVar('table-bg-color');
  font-size: 14px;
……

不止table，还有一些其他组件的特性，为什么不考虑都定义在common/var中？

### Additional comments

_(empty)_

**Comment by gimjin** [comment]
el-table-column 并没有 size prop 属性，el-config-provider 全局设置 size 还是实验性功能未来应该是会支持全局配置。
现阶段可以用 el-text 控制，像乐高一样。
